src/popup/searchModule.js

```diff
@@ -1,13 +1,20 @@
 import { elements } from './base';
-import { unicodeToString, providerLogos, addLoadMoreButton } from './helper';
+import {
```

Comment:
Undo these changes. Your editor must be wrapping the lines on save.

Reply:
Okay, On it